auction-server/src/opportunity/repository/models.rs

```diff
@@ -50,7 +50,9 @@ pub struct OpportunityMetadataEvm {
 #[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
 pub struct OpportunityMetadataSvmProgramLimo {
     #[serde_as(as = "Base64")]
-    pub order: Vec<u8>,
+    pub order:         Vec<u8>,
+    #[serde_as(as = "DisplayFromStr")]
+    pub order_address: Pubkey,
```

Comment:
What will happen to old opps? Maybe create a migration to remove old svm opps?
If there is no opps yet, we are good to go

Reply:
no opps on per-staging